### DISCUSSION TITLE
MTV East

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#731] [Show and tell]
**Channel Name:** MTV
**Channel ID:** MTVEast.us
**Country:** United States
**Language:** English
**Website:** xxx
**Stream URL (optional):** https://www.mtv.com/live-tv/xzq4o4
**Notes (optional):** There's a stream in the link above.

**Comment by BellezaEmporium** [comment]
Most certainly PPV stream.

Needs an US IP with US ISP credentials to get in.

Asks for authorization (certainly based on IP) using https://auth.mtvnservices.com/access/authorization/mtv?clientId=mtv-web&countryCode=us

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/bz_nexgen.m3u#L14-L15
